guides/image-sources/prov-first-local-boot-pxe.iuml

```diff
@@ -0,0 +1,15 @@
+Host -> DHCP : requests the reserved IP and DHCP options
+Host -> TFTP : downloads bootloader
+Host -> TFTP : gets  MAC-based bootloader config
+Host -> Host : config tells bootloader to boot from HDD
+Host -> Host : boots from HDD
+!if ($puppet)
```

Comment:
I am just curious: Where does the `$puppet` come from? Could we also have `$katello` to skip/add some steps in the diagrams? Or would that be too complicated?

Reply:
The original diagrams by lzap were very Puppet centric. I wasn't sure if we want to remove it altogether.
It is possible to only include katello stuff conditionally, that should be easy.

Reply:
I've edited the diagrams to be neutral towards the source of installation media.